com.flashforge.FlashPrint.json

```diff
@@ -0,0 +1,80 @@
+{
+  "app-id": "com.flashforge.FlashPrint",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "FlashPrint",
+  "tags": [
+    "proprietary"
+  ],
+  "finish-args": [
+    "--filesystem=home",
+    "--socket=x11",
+    "--device=dri",
+    "--share=ipc"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "FlashPrint",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D apply_extra /app/bin/apply_extra",
+        "install -D flashprint-bin /app/bin/FlashPrint",
+        "install -Dm644 com.flashforge.FlashPrint.appdata.xml /app/share/appdata/com.flashforge.FlashPrint.appdata.xml",
+        "install -D /usr/bin/ar /app/bin/ar",
+        "mkdir -p /app/lib",
+        "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib"
+      ],
+      "sources": [
+        {
+          "type": "script",
+          "dest-filename": "apply_extra",
+          "commands": [
+            "ar p flashprint.deb data.tar.xz | tar xJ",
+            "rm -f flashprint.deb",
+            "mv usr/* .",
+            "rmdir usr",
+            "mkdir -p export/share/applications",
+            "sed s/Icon=flashforge/Icon=com.flashforge.FlashPrint share/applications/FlashPrint.desktop > export/share/applications/com.flashforge.FlashPrint",
```

Comment:
1. Invalid sed command. Should be:
```
sed 's/Icon=flashforge/Icon=com.flashforge.FlashPrint/'
```
or
```
sed 's/(Icon=).+/\1com.flashforge.FlashPrint/'
```
2. You can (and should) use `desktop-file-edit` or `desktop-file-install` instead of sed.
You can also use `rename-icon` and (if needed) `copy-icon`.